### PR TITLE
Define and activate release v12.5.0 for AWS

### PR DESCRIPTION
**Pull request by alex-dabija**
AWS v12.5.0 Release contains the Network Pools feature.

### CODE REVIEW

**Comment by njuettner** [comment]
Do we want to include Kubernetes 1.17.13? This fixes 2 CVE's (CVE-2020-8566, CVE-2020-8564). One is about Secret leaks in logs for vSphere Provider kube-controller-manager and the other one is  Incomplete fix for CVE-2019-11250 allows for token leak in logs when logLevel >= 9.

**Comment by alex-dabija** [comment]
We also included Kubernetes v1.17.13 in the release.

**Review by paurosello — APPROVED** [review]

File: aws/v12.5.0/release.diff [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    version: 0.1.0                                                     version: 0.1.0
+  - name: cluster-operator                                           - name: cluster-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 3.2.0                                              |      version: 3.3.1
+  - name: containerlinux                                             - name: containerlinux
+    version: 2512.5.0                                                  version: 2512.5.0
+  - name: etcd                                                       - name: etcd
+    version: 3.4.13                                                    version: 3.4.13
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.17.12                                                   version: 1.17.12
```
Comment (njuettner) [inline aws/v12.5.0/release.diff:65]: diff is missing 1.17.13

**Review by njuettner — CHANGES_REQUESTED** [review]

File: aws/v12.5.0/release.diff (hunk 1 above)
Comment (alex-dabija) [reply aws/v12.5.0/release.diff]: Good catch! I regenerated the release.diff file.

**Review by njuettner — APPROVED** [review]
lgtm 🚀

**Pull request merged**